next-cloudinary/src/plugins/cropping.js

```diff
@@ -8,21 +8,21 @@ export function plugin({ cldImage, options, cldOptions } = {}) {
 
   let transformationString = `c_${crop},w_${width}`;
 
-  if ( !gravity && cropsGravityAuto.includes(crop) ) {
+  if ( !gravity ) {
+    if ( cropsGravityAuto.includes(crop) && !gravity ) {
     gravity = 'auto';
+    } else {
```

Comment:
you added this as an else related to the gravity configurations, but this isn't related to gravity. this is likely the cause of the regression

```
if ( !['limit'].includes(crop) ) {
    transformationString = `${transformationString},h_${height}`;
  }
```

that likely shouldn't be change at all

Reply:
I have added the Command again to the commit!